src/lib.rs

```diff
@@ -275,6 +275,8 @@
 // an unstable crate
 #![cfg_attr(rustbuild, feature(staged_api, rustc_private))]
 #![cfg_attr(rustbuild, unstable(feature = "rustc_private", issue = "27812"))]
+// TODO: Remove once https://github.com/rust-lang/rust/pull/72488 is merged
```

Comment:
I'll wait for https://github.com/rust-lang/rust/pull/72488 before merging this PR so this unstable feature can be removed.